contracts/chain-adapters/Arbitrum_CustomGasToken_Adapter.sol

```diff
@@ -155,16 +155,16 @@ contract Arbitrum_CustomGasToken_Adapter is AdapterInterface, CircleCCTPAdapter
     // ticket’s calldata in the retry buffer. (current base submission fee is queryable via
     // ArbRetryableTx.getSubmissionPrice). ArbRetryableTicket precompile interface exists at L2 address
     // 0x000000000000000000000000000000000000006E.
-    // The Arbitrum Inbox requires that this uses 18 decimal precision.
+    // The Arbitrum Inbox requires that this uses the same precision as the L2's native gas token.
     uint256 public immutable L2_MAX_SUBMISSION_COST;
 
-    // L2 Gas price bid for immediate L2 execution attempt (queryable via standard eth*gasPrice RPC)
-    // The Arbitrum Inbox requires that this is specified in gWei (e.g. 1e9 = 1 gWei)
+    // L2 Gas price bid for immediate L2 execution attempt (queryable via standard eth_gasPrice RPC)
+    // Since this price fluctuates, it should be set to a conservative value. The gas price must be specified in Wei.
```

Comment:
i don't think we should change this comment

Reply:
It's been switched back https://github.com/across-protocol/contracts/pull/699/commits/6437f7d55cc9c274576350617b28cbb08bb65c10